internal/cmd/skupper/connector/connector.go

```diff
@@ -164,3 +163,42 @@ func CmdConnectorDeleteFactory(configuredPlatform types.Platform) *cobra.Command
 
 	return cmd
 }
+
+func CmdConnectorGenerateFactory(configuredPlatform types.Platform) *cobra.Command {
+	kubeCommand := kube.NewCmdConnectorGenerate()
+	nonKubeCommand := nonkube.NewCmdConnectorGenerate()
+
+	cmdConnectorGenerateDesc := common.SkupperCmdDescription{
+		Use:   "generate <name> <port>",
+		Short: "Generate a connector",
```

Comment:
Suggestion: The `short` description should be: "Generate a connector resource"

Reply:
done